spring-cloud-kubernetes-fabric8-config/src/test/java/org/springframework/cloud/kubernetes/fabric8/config/locator_retry/fail_fast_enabled_retry_disabled/ConfigDataConfigFailFastEnabledButRetryDisabled.java

```diff
@@ -49,7 +51,10 @@ class ConfigDataConfigFailFastEnabledButRetryDisabled extends ConfigFailFastEnab
 	private static KubernetesClient mockClient;
 
 	@MockBean
-	KubernetesNamespaceProvider kubernetesNamespaceProvider;
+	private KubernetesNamespaceProvider kubernetesNamespaceProvider;
+
+	@SpyBean
```

Comment:
we need this `@SpyBean` because the test itself has this:

```
@SpyBean
private Fabric8ConfigMapPropertySourceLocator propertySourceLocator;
```

and this one has a dependency on `ConfigMapConfigProperties` in its constructor, but since we have changed the code to not provide these properties if configmap is disabled (which we do in this test), we need to provide it.